docs/configuration/README.md

```diff
@@ -267,8 +267,16 @@ The `ingester_config` block configures Ingesters.
 # period as long as they receieve no further activity.
 [chunk_idle_period: <duration> | default = 30m]
 
-# The maximum size in bytes a chunk can be before it should be flushed.
+# The maximum _uncompressed_ size in bytes of a chunk block, 
```

Comment:
I don't think this is 100% accurate: almost all blocks are going to end up being greater than this size. This is more like the cuttoff point for when an uncompressed block should be compressed. 

Do you think it's worth explaining that in the docs?

Reply:
yeah agree, will reword this

Reply:
@rfratto reworded, let me know what you think

Reply:
Nice, this is more clear. Thanks!